src/components/Navbar.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react'
 import styled from 'styled-components'
 import { Navbar, Nav, Button } from 'react-bootstrap'
+import { AuthContext } from '../context/Auth'
 
 export default (props) => {
+  const authValue = React.useContext(AuthContext)
+  const handleLogOut = authValue.logOut
   return (
     <CustomNavbar sticky='top' collapseOnSelect expand='md'>
       <Navbar.Brand href='/' style={{ color: '#30323D', fontSize: '1.9rem' }}>SG VERIFACT</Navbar.Brand>
       <Navbar.Toggle aria-controls='responsive-navbar-nav' />
       <Navbar.Collapse id='responsive-navbar-nav' style={{ backgroundColor: 'white' }}>
         <Nav className='mr-auto' />
         <Nav style={{ alignItems: 'center' }}>
-          <Nav.Link style={{ color: '#30323D' }} href='login'>Log In</Nav.Link>
-          <Nav.Link style={{ color: '#30323D' }} href='signup'>
-            Sign Up
-          </Nav.Link>
+          {authValue.isLoggedIn
+            ? (<Nav.Link style={{ color: '#30323D' }} onClick={handleLogOut}>Log Out</Nav.Link>)
```

Comment:
hex color should not be inline. Use styled components as these all share the same styles.